packages/fxa-settings/src/components/FormPhoneNumber/index.stories.tsx

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import FormPhoneNumber from '.';
+import { withLocalization } from 'fxa-react/lib/storybooks';
+import { Meta } from '@storybook/react';
+import AppLayout from '../AppLayout';
+
+export default {
+  title: 'Components/FormPhoneNumber',
+  component: FormPhoneNumber,
+  decorators: [withLocalization],
+} as Meta;
+
+export const Default = () => (
+  <AppLayout>
+    <FormPhoneNumber localizedCTAText="Send code" />
+  </AppLayout>
+);
+
+export const WithInfo = () => (
+  <AppLayout>
+    <FormPhoneNumber showInfo localizedCTAText="Confirm" />
```

Comment:
We will need a function prop to use/verify/validate/etc. the phone number in a higher-level component, and maybe we pass in the info banner content props so banner content can be configured by the parent? (In case we end up using different message in different flows)

Reply:
That's what I pictured as well. 👍 I figured we'd update that with another ticket though since this one is just "build the input component" and I've got the data (phone number) logging in the form component? The `showInfo` bit was meant to be a placeholder or adjusted as needed.

Reply:
Yeah, that works for me! It's quick to add/update in my ticket